acestep/ui/gradio/interfaces/generation_tab_simple_controls.py

```
@@ -0,0 +1,118 @@
+            with gr.Column(scale=1, min_width=80, elem_classes="icon-btn-wrap"):
+                random_desc_btn, _ = build_simple_action_controls(
+                    create_random_desc=True,
+                    create_sample=False,
+                )
+        with gr.Row(equal_height=True):
+            _, create_sample_btn = build_simple_action_controls(
+                create_random_desc=False,
+                create_sample=True,
+            )
+    if random_desc_btn is None or create_sample_btn is None:
+        raise RuntimeError("Simple action controls were not created as expected.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)